core/src/retransmit_stage.rs

```diff
@@ -6,7 +6,7 @@ use crate::cluster_info::{
     compute_retransmit_peers, ClusterInfo, DATA_PLANE_FANOUT, GROW_LAYER_CAPACITY,
     NEIGHBORHOOD_SIZE,
 };
-use crate::packet::SharedBlob;
+use crate::packet::Blob;
```

Comment:
Investigate if removing `SharedBlob` will cause unnecessary copies in this module